### DISCUSSION TITLE
How to visualize the created mesh by meshio

### DISCUSSION THREAD

**Discussion by alisheikholeslam** [nschloe/meshio#1326] [Q&A]
I have created a triangle mesh. Is there any modules in meshio that shows the created mesh in 3d on windows? If not what is the best way?
```
import meshio

points = [[50,50,50], [70,40,50], [50,40,80], [80,70,50]]
cells = [("triangle", [[0,1,2], [2,1,3], [1,0,3]])]
triangle_colours = [[0.46, 0.13, 0.26, 1], [0.5, 0.18, 0.28, 1], [0.92, 0.8, 0.71, 1]]

mesh = meshio.Mesh(points, cells, cell_data={"a": [triangle_colours]},)
```

**Comment by nschloe** [comment]
I have something for this lined up for a release. Will post here.

**Reply by alisheikholeslam** [reply]
Thanks. Do you post before the new release or I should wait for the new release.

**Reply by nschloe** [reply]
Best wait for the release.

**Reply by alisheikholeslam** [reply]
Ok Thanks. I will wait. Any date range for the release time (How long it takes for new release, known?)?

**Reply by nschloe** [reply]
Nope.
